src/lxc/confile.c

```diff
@@ -1778,6 +1787,15 @@ static int set_config_mount_auto(const char *key, const char *value,
 
 		lxc_conf->auto_mounts &= ~allowed_auto_mounts[i].mask;
 		lxc_conf->auto_mounts |= allowed_auto_mounts[i].flag;
+		if (is_shmounts) {
+			lxc_conf->lxc_shmount.path_host =
+					strdup(token + (sizeof("shmounts:") - 1));
+			if (!strcmp(lxc_conf->lxc_shmount.path_host, "")) {
+				ERROR("Invalid shmounts path: empty");
+				break;
```

Comment:
Hm, it would be better to verify that this isn't empty before the `strdup()`. You can use:

```
bool lxc_config_value_empty(const char *value)
{
	if (value && strlen(value) > 0)
		return false;

	return true;
}
```
This is the simple verification we need right now.